frontend/src/lib/components/ServiceRequestPreview.svelte

```diff
@@ -0,0 +1,107 @@
+<script lang='ts'>
+	import messages from '$media/messages.json'
+	import { goto } from '$app/navigation';
+	import { Badge, Card } from 'stwui';
+	import { Dropdown } from 'stwui';
+	import type { GetServiceRequestsResponse, ServiceRequest } from '$lib/services/Libre311/Libre311';
+	import dropDownIcon from '$lib/assets/ellipsis-vertical.svg';
+
+	export let serviceRequest: GetServiceRequestsResponse;
+
+	let visible: Boolean = false;
+
+	function closeDropdown() {
+		visible = false;
+	}
+
+	function toggleDropdown() {
+		visible = !visible;
+	}
+
+	function getStatus (serviceRequest: ServiceRequest) : string  {
+		switch (serviceRequest.status) {
+			case 'closed': {
+				return 'success';
+			}
+			case 'open': {
+				return 'warn';
+			}
+		}
+		return 'error';
+	}
+
+	function toTimeStamp (serviceRequest: ServiceRequest) {
+		return `${new Date(serviceRequest.requested_datetime).toLocaleDateString()} ${new Date(serviceRequest.requested_datetime).toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'})}`
+	}
+
+	const gotoServiceRequest = () => {
+		goto(`/issues/map/${serviceRequest.service_request_id}`);
+	}
+</script>
+
+<Card>
+	<div class="my-1 mx-4" slot="content">
+
+		<div class='flow-root'>			
+			<h2 class='text-base float-left tracking-wide'>
+				#{serviceRequest.service_request_id}
+			</h2>
+			<Badge class='text-sm float-right' type={getStatus(serviceRequest)}>{serviceRequest.status}</Badge>
+		</div>
+
+		<p class='text-sm font-extralight my-1'>{toTimeStamp(serviceRequest)} </p>
+		
+		{#if serviceRequest.media_url}
+			<div class='bg-[#D9D9D9] rounded-md'>
+				<button class='flex overflow-y-hidden overflow-x-scroll mx-auto rounded-md' on:click={gotoServiceRequest}>
```

Comment:
Instead of using `on:click` and calling the `goto` function, it is better to just use the `<a>` tag.  That way the user can hover over and see where the click action will take them. In this [article]( https://css-tricks.com/use-button-element/#aa-when-links-make-more-sense) mention a good rule of thumb is, if there is a meaningful href that can be used then you should use an  `<a>` tag.